lib/classes/video.js

```diff
@@ -791,6 +816,7 @@ class Video {
                 if (stream.codec_name !== 'flac' || _self.options.forceHeAudio) {
                     _self.encoder.logger.verbose('Audio stream', colors.yellow(helpers.getStreamTitle(stream) + ' (index: ' + stream.index + ')'), 'will be encoded to HE Audio.');
                     _self.ffmpegCommand.outputOptions('-c:a:' + i, 'libopus');
+                    _self.ffmpegCommand.outputOptions('-af', "aformat=channel_layouts='7.1|5.1|stereo'");
```

Comment:
this needs to be:
`_self.ffmpegCommand.audioFilters("aformat=channel_layouts='7.1|5.1|stereo'");`
I _think_, because you can only have one `-af` or it will ignore the rest.